crud/example_test.go

```diff
@@ -78,6 +78,43 @@ func ExampleResult_rowsCustomType() {
 	// [{{} 2010 45 bla}]
 }
 
+func ExampleResult_operationDataCustomType() {
+	conn := exampleConnect()
+	req := crud.MakeInsertManyRequest(exampleSpace).Tuples([]interface{}{
+		[]interface{}{uint(1), 1, "bla"},
+		[]interface{}{uint(1), 2, "blabla"},
+		[]interface{}{uint(2011), 3, nil},
+	})
+
+	type Tuple struct {
+		_msgpack struct{} `msgpack:",asArray"` //nolint: structcheck,unused
+		Id       uint64
+		BucketId uint64
+		Name     string
+	}
+
+	ret := crud.MakeResult(reflect.TypeOf(Tuple{}))
```

Comment:
Please, add an example with a default crud.Result{} (without a decoding type).

Reply:
Added.

Reply:
Let's make examples on the same/similart values and for the same requests (InsertObjectManyRequest), to make it is obvious what the difference is.

Reply:
Sure, done.